sdk/appconfiguration/app-configuration/src/generated/src/appConfigurationContext.ts

```diff
@@ -1,51 +1,63 @@
 /*
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License. See License.txt in the project root for
- * license information.
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
  *
  * Code generated by Microsoft (R) AutoRest Code Generator.
- * Changes may cause incorrect behavior and will be lost if the code is
- * regenerated.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
  */
 
 import * as coreHttp from "@azure/core-http";
-import * as Models from "./models";
+import { ApiVersion10, AppConfigurationOptionalParams } from "./models";
+
+const packageName = "app-configuration";
+const packageVersion = "1.1.1";
```

Comment:
Just to check - there's some logic in the build that submits a PR to update the version number in the package.json and (I believe) in source files as well. Do you know if this change will break that?

Reply:
As of today, the version updater updates the version in custom layer only. The master branch code does not have any fixed version in the generated layer - https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/appconfiguration/app-configuration/src/generated/src

So, this new field within the generated layer will not be updated in the future. But, the one in the custom layer has not been changed and will be updated. 

If you prefer, I can file an issue under version-updater to update this field also.

Reply:
@richardpark-msft Made the configuration change at https://github.com/Azure/azure-sdk-for-js/pull/13095/files#diff-a2820367f311281552f23cbd1d81de6e2e015b7f0d001a1d4118452b44ee9652R82 which will ensure the Version Updater tool will ensure that this value will also be updated in the future.